src/Components/HoverView.re

```diff
@@ -0,0 +1,152 @@
+open Revery;
```

Comment:
Nice! Thanks for refactoring this out to a common place - good chunk of code to share between hover and signature help 👍